1.0/learn/installing-ballerina.md

```diff
@@ -10,28 +10,25 @@ redirect_from:
 
 # Installing Ballerina
 
-- [Downloading the Ballerina distribution](#downloading-the-ballerina-distribution)
+- [Prerequisites](#prerequisites)
 - [Installing Ballerina via installers](#installing-ballerina-via-installers)
 - [Installing from source](#installing-from-source)
 - [Uninstalling Ballerina](#uninstalling-ballerina)
 - [Getting help](#getting-help)
 - [What's next](#what's-next)
 
-## Downloading the Ballerina distribution
+## Prerequisites
 
-[Download](https://ballerina.io/downloads/) a Ballerina distribution based on your operating system.
-
-> Ballerina binary distributions are available for the following supported operating systems and architectures. Ensure that your system meets the below requirements before you proceed with the installation.
+Ballerina binary distributions are available for the following supported operating systems and architectures. Ensure that your system meets the below requirements before you proceed with the installation.
 - Windows Vista SP2 x64 or later
 - Ubuntu Linux 12.04 x64 - LTS and above
 - OS X 10.8.3 x64 and above
```

Comment:
I think we need to revisit this list of supported OS. Some concerns:
- Vista is no longer supported by Microsoft (ended in 2017 apparently).
- Ubuntu is not the only linux OS we support.

Reply:
@pubudu91 Can you create a new GitHub issue with the updated list that should be included?

Reply:
will do

Reply:
Thanks a lot!

Reply:
https://github.com/ballerina-platform/ballerina-dev-website/issues/591